lib/keystores/jks/pkcs8_key.rb

```diff
@@ -60,8 +60,10 @@ def encode_private_key
     class RSA
       original_initialize = instance_method(:initialize)
 
-      define_method(:initialize) do |der_or_pem, pass_phrase = nil|
+      define_method(:initialize) do |der_or_pem = nil, pass_phrase = nil|
```

Comment:
Overriding built-in C methods from the Ruby stdlib? What could possibly go wrong

Reply:
As I commented in the commit, this still doesn't match the full behavior of the normal constructor (which takes either a number as the size, or some file/string object as the key) but I don't want to touch this any more than strictly necessary